samples/system-test/spanner.test.js

```diff
@@ -41,277 +41,298 @@ const spanner = new Spanner({
   projectId: PROJECT_ID,
 });
 
-test.before(tools.checkCredentials);
-
-test.before(async () => {
-  const instance = spanner.instance(INSTANCE_ID);
-  const database = instance.database(DATABASE_ID);
-
-  try {
-    await instance.delete();
-  } catch (err) {
-    // Ignore error
-  }
-
-  try {
-    await database.delete();
-  } catch (err) {
-    // Ignore error
-  }
-
-  const [, operation] = await instance.create({
-    config: 'regional-us-central1',
-    nodes: 1,
-    labels: {
-      'gcloud-sample-tests': 'true',
-    },
+describe('Spanner', () => {
+  before(() => tools.checkCredentials);
+
+  before(async () => {
+    const instance = spanner.instance(INSTANCE_ID);
+    const database = instance.database(DATABASE_ID);
+
+    try {
+      await instance.delete();
+    } catch (err) {
+      // Ignore error
+    }
+
+    try {
+      await database.delete();
+    } catch (err) {
+      // Ignore error
+    }
+
+    const [, operation] = await instance.create({
+      config: 'regional-us-central1',
+      nodes: 1,
+      labels: {
+        'gcloud-sample-tests': 'true',
+      },
+    });
+
+    await operation.promise();
   });
 
-  await operation.promise();
-});
-
-test.before(async () => {
-  const [instances] = await spanner.getInstances({
-    filter: 'labels.gcloud-sample-tests:true',
-  });
+  before(async () => {
+    const [instances] = await spanner.getInstances({
+      filter: 'labels.gcloud-sample-tests:true',
+    });
 
-  instances.forEach(async instance => {
-    const {operations} = await getOperations(instance.metadata.name);
+    instances.forEach(async instance => {
+      const {operations} = await getOperations(instance.metadata.name);
 
-    operations
-      .filter(operation => {
-        return operation.metadata['@type'].includes('CreateInstance');
-      })
-      .filter(operation => {
-        const yesterday = new Date();
-        yesterday.setHours(-24);
+      operations
+        .filter(operation => {
+          return operation.metadata['@type'].includes('CreateInstance');
+        })
+        .filter(operation => {
+          const yesterday = new Date();
+          yesterday.setHours(-24);
 
-        const instanceCreated = new Date(operation.metadata.startTime);
+          const instanceCreated = new Date(operation.metadata.startTime);
 
-        return instanceCreated < yesterday;
-      })
-      .forEach(async () => await instance.delete());
+          return instanceCreated < yesterday;
+        })
+        .forEach(async () => await instance.delete());
+    });
   });
-});
 
-test.after.always(async () => {
-  const instance = spanner.instance(INSTANCE_ID);
-  const database = instance.database(DATABASE_ID);
-
-  try {
-    await database.delete();
-  } catch (err) {
-    // Ignore error
-  }
-
-  try {
-    await instance.delete();
-  } catch (err) {
-    // Ignore error
-  }
-});
+  afterEach(async () => {
```

Comment:
I think you want `after(...` here, not afterEach

Reply:
Yes you are right. I will update in next commit.